### PR TITLE
Completed Subscriptions section of Kaui Guide

### PR DESCRIPTION
**Pull request by ghost**
I don't know if I got all of this right, so keep a sharp eye out. :-0 

There's a PDF available if you want to read it that way. See page 35 - 41.

### CODE REVIEW

File: userguide/kaui/Kaui_Guide_Draft.adoc [hunk 1]
```diff
@@ -933,31 +933,228 @@ To delete a payment method:
...
+
+|===
+
+==== Add an Add-On Product to a Subscription
+
+This section explains how to add an add-on product to a customer's subscription.
+
+[NOTE]
+*Note:* To add an add-on to a subscription, the add-on must exist in the catalog. For information on adding an add-on to the catalog, see <<part_6_tenants_and_admin>>.
+
```
Comment (reshmabidikar) [inline userguide/kaui/Kaui_Guide_Draft.adoc:1004]: Minor correction: The add-on must exist in the catalog and be available in the base plan.

File: userguide/kaui/Kaui_Guide_Draft.adoc [hunk 2]
```diff
@@ -933,31 +933,228 @@ To delete a payment method:
...
+. In the *New plan* field, select a different plan.
+. If you need to override the price for the plan you've selected, enter the amount in the *Price Override* field.
+. To accept the default policy for when this change takes effect, leave *Default policy* selected and click *Save.*
+
+OR
+
+Use one of the other two options – *Specify Policy* or *Specify a date* – to alter when this change takes effect:
+
+* *Specify Policy*: You can select either `IMMEDIATE` or `END_OF_TERM`. For information on these two policies, see the https://docs.killbill.io/latest/userguide_subscription.html#\_subscription_alignment_rules["Subscription Alignment Rules"] section of the \_Subscription Guide_.
+
```
Comment (reshmabidikar) [inline userguide/kaui/Kaui_Guide_Draft.adoc:1097]: Might be worth mentioning that these rules can be specified in the catalog under the [plan change timing section](https://docs.killbill.io/latest/userguide_subscription.html#_plan_change_timing) but the values specified here override the catalog values.

File: userguide/kaui/Kaui_Guide_Draft.adoc [hunk 3]
```diff
@@ -933,31 +933,228 @@ To delete a payment method:
...
+| Cancel start of term (full credit)
+| Cancels the subscription immediately and refunds whatever amounts have been paid toward the subscription.
+
+| Cancel immediately (partial credit)
+| Cancels the subscription immediately and applies a partial credit to the account based on how much of the service has been consumed.
+
+| Cancel end of term (no credit)
+| Cancels the subscription immediately with no refund to the customer. Note that after using this option to cancel, you can still reinstate the customer (i.e., reverse the cancellation).
+
```
Comment (reshmabidikar) [inline userguide/kaui/Kaui_Guide_Draft.adoc:1134]: Is there a typo here? Shouldn't this be `Cancels the subscription at the end of the billing period`?

File: userguide/kaui/Kaui_Guide_Draft.adoc [hunk 4]
```diff
@@ -933,31 +933,228 @@ To delete a payment method:
...
+Kaui opens the Change Subscription screen:
+
+image::ChangeSubscription.png[width=450,align="center"]
+
+[start=3]
+. In the *New plan* field, select a different plan.
+. If you need to override the price for the plan you've selected, enter the amount in the *Price Override* field.
+. To accept the default policy for when this change takes effect, leave *Default policy* selected and click *Save.*
+
```
Comment (reshmabidikar) [inline userguide/kaui/Kaui_Guide_Draft.adoc:1091]: Should we mention what the default policy is? AFAIK, it is `END_OF_TERM` but I'll let @pierre or @sbrossie confirm.

File: userguide/kaui/Kaui_Guide_Draft.adoc [hunk 5]
```diff
@@ -933,31 +933,228 @@ To delete a payment method:
...
+Otherwise, if you leave the checkbox empty, the entitlement date is the same as you just selected BUT the billing date defaults to _immediate_.
+
+|===
+
+==== Update the Billing Cycle Day
+
+Follow the steps below to update the billing cycle day (BCD) defined for a subscription. These steps assume the subscription is billed on a monthly basis.
+
+For information on how changing the BCD affects the subscription, see the https://docs.killbill.io/latest/userguide_subscription.html#\_billing_alignment_rules["Billing Alignment Rules"] section of the \_Subscription Guide_.
+
```
Comment (reshmabidikar) [inline userguide/kaui/Kaui_Guide_Draft.adoc:1151]: The [Billing Alignment Rules](https://docs.killbill.io/latest/userguide_subscription.html#_billing_alignment_rules) specify whether billing occurs at the account/subscription or bundle level. Updating a subscription BCD in Kaui results in the system using the new BCD irrespective of what is specified in the billing alignment rules in the catalog. In other words, the BCD specified in Kaui overrides the catalog billing alignment rules.  Might be useful to add this context here.

File: userguide/kaui/Kaui_Guide_Draft.adoc [hunk 6]
```diff
@@ -933,31 +933,228 @@ To delete a payment method:
...
+| If you need to override the price for the add-on you've selected, enter an amount in this field.
+
+| Immediate Creation
+| Select this option to start the subscription immediately.
+
+| Specify a date
+| To define a specific date on which the subscription begins, select this option, and choose a date from the *Date* field that appears.
+
+|===
+
```
Comment (reshmabidikar) [inline userguide/kaui/Kaui_Guide_Draft.adoc:1039]: Below the "Add Add-on", there are other options like "Transfer Ownership" and "Pause/Resume". Are these intentionally left out?

File: userguide/kaui/Kaui_Guide_Draft.adoc (hunk 4 above)
Comment (ghost) [reply userguide/kaui/Kaui_Guide_Draft.adoc:1091]: Added to line 1042 (just general info) and addressed END_OF_TERM in line 1058.

File: userguide/kaui/Kaui_Guide_Draft.adoc (hunk 2 above)
Comment (ghost) [reply userguide/kaui/Kaui_Guide_Draft.adoc:1097]: Edited - see line 1062

File: userguide/kaui/Kaui_Guide_Draft.adoc (hunk 3 above)
Comment (ghost) [reply userguide/kaui/Kaui_Guide_Draft.adoc:1134]: You are great! That was an error. Fixed - line 1101

File: userguide/kaui/Kaui_Guide_Draft.adoc (hunk 1 above)
Comment (ghost) [reply userguide/kaui/Kaui_Guide_Draft.adoc:1004]: Fixed - see line 1003

File: userguide/kaui/Kaui_Guide_Draft.adoc (hunk 5 above)
Comment (ghost) [reply userguide/kaui/Kaui_Guide_Draft.adoc:1151]: Removed reference to Billing Alignment Rules. Added your info to line 118.

File: userguide/kaui/Kaui_Guide_Draft.adoc (hunk 6 above)
Comment (ghost) [reply userguide/kaui/Kaui_Guide_Draft.adoc:1039]: See the ALL NEW Pause/Resume section below (1040). Pierre said to not document the Transfer Ownership feature for now.

**Review by ghost — COMMENTED** [review]
@reshmabidikar - All of your comments have been addressed. Let me know if I missed anything. Thanks for catching all of that.

File: userguide/kaui/Kaui_Guide_Draft.adoc [hunk 7]
```diff
@@ -933,31 +933,262 @@ To delete a payment method:
...
+[start=3]
+. In the *New plan* field, select a different plan.
+. If you need to override the price for the plan you've selected, enter the amount in the *Price Override* field.
+. To accept the system's default policy for changing subscriptions (`END_OF_TERM`), leave *Default policy* selected and click *Save.*
+
+OR
+
+You can select one of two other options: *Specify Policy* or *Specify a date*. This will override the policy defined in the catalog for the plan. (For information on this topic, see see https://docs.killbill.io/latest/userguide_subscription.html#_plan_change_timing["Plan Change Timing"] in the _Subscription Guide_.)
+
```
Comment (reshmabidikar) [inline userguide/kaui/Kaui_Guide_Draft.adoc:1095]: The URL is not rendered properly in the asciidoc file. See https://github.com/FlowWriter/killbill-docs/blob/kaui-only-5/userguide/kaui/Kaui_Guide_Draft.adoc#change-a-subscription

**Review by reshmabidikar — COMMENTED** [review]
All the changes look good @FlowWriter, I've found one minor new issue about URLs not rendering properly in the asciidoc file. I've added a comment related to this.

File: userguide/kaui/Kaui_Guide_Draft.adoc [hunk 8]
```diff
@@ -933,31 +933,262 @@ To delete a payment method:
...
+image::ChangeSubscription.png[width=450,align="center"]
+
+[start=3]
+. In the *New plan* field, select a different plan.
+. If you need to override the price for the plan you've selected, enter the amount in the *Price Override* field.
+. To accept the system's default policy for changing subscriptions (`END_OF_TERM`), leave *Default policy* selected and click *Save.*
+
+OR
+
+You can select one of two other options: *Specify Policy* or *Specify a date*. This will override the policy defined in the catalog for the plan. (For information on this topic, see see https://docs.killbill.io/latest/userguide_subscription.html#_plan_change_timing["Plan Change Timing"] in the _Subscription Guide_.)
```
Comment (pierre) [inline userguide/kaui/Kaui_Guide_Draft.adoc:1094]: Nit: `see see`

**Comment by pierre** [comment]
Merging so we can check the rendering.

**Pull request merged**